From 5ccb5386304c26f25b0a9eb10ce9edb6da32f91a Mon Sep 17 00:00:00 2001
From: Julien Moutinho <julm+srht@sourcephile.fr>
Date: Sat, 12 Feb 2022 00:11:59 +0100
Subject: [PATCH git.sr.ht] Revert "Add webhook queue monitoring"

This reverts commit 7ea630b776947ab82438d0ffa263b0f9d33ebff3.

Which has broken Unix socket support for Redis.
See https://lists.sr.ht/~sircmpwn/sr.ht-dev/%3C20211208082636.65665-1-me%40ignaskiela.eu%3E#%3C20211216033723.wefibfulfjhqnhem@sourcephile.fr%3E
---
 gitsrht/app.py      | 3 ---
 gitsrht/webhooks.py | 5 +----
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/gitsrht/app.py b/gitsrht/app.py
index e9ccb56..4928851 100644
--- a/gitsrht/app.py
+++ b/gitsrht/app.py
@@ -48,9 +48,6 @@ class GitApp(ScmSrhtFlask):
         self.add_template_filter(url_quote)
         self.add_template_filter(commit_links)
 
-        from gitsrht.webhooks import webhook_metrics_collector
-        self.metrics_registry.register(webhook_metrics_collector)
-
         @self.context_processor
         def inject():
             notice = session.get("notice")
diff --git a/gitsrht/webhooks.py b/gitsrht/webhooks.py
index 8a203fe..6240d50 100644
--- a/gitsrht/webhooks.py
+++ b/gitsrht/webhooks.py
@@ -7,13 +7,10 @@ if not hasattr(db, "session"):
     db.init()
 from srht.webhook import Event
 from srht.webhook.celery import CeleryWebhook, make_worker
-from srht.metrics import RedisQueueCollector
 from scmsrht.webhooks import UserWebhook
 import sqlalchemy as sa
 
-webhook_broker = cfg("git.sr.ht", "webhooks")
-worker = make_worker(broker=webhook_broker)
-webhook_metrics_collector = RedisQueueCollector(webhook_broker, "srht_webhooks", "Webhook queue length")
+worker = make_worker(broker=cfg("git.sr.ht", "webhooks"))
 
 class RepoWebhook(CeleryWebhook):
     events = [
-- 
2.34.1

